test/query-test.ts

```diff
@@ -0,0 +1,89 @@
+import axios from 'axios';
+import { prisma, serverUrl } from './server-setup-test.js';
+import jwt from 'jsonwebtoken';
+import { expect } from 'chai';
+
+let testToken: string;
+
+const query = {
+  query: `query getUsers {
+    users{
+    id
+    name
+    email
+    birthDate
+    }
+  }`,
+};
+
+describe('Queries Test', function () {
+  before(async () => {
+    await prisma.user.deleteMany();
+    await prisma.user.createMany({
+      data: [
+        {
+          name: 'usuario1',
+          email: 'usuario1@example.com',
+          password: '45687a',
+          birthDate: '2003-01-01',
+        },
+        {
+          name: 'usuario2',
+          email: 'usuario2@example.com',
+          password: '45687a',
+          birthDate: '2003-01-01',
+        },
+        {
+          name: 'usuario3',
+          email: 'usuario3@example.com',
+          password: '45687a',
+          birthDate: '2003-01-01',
+        },
+      ],
+    });
+    testToken = jwt.sign({ id: 1 }, process.env.SECRET_KEY ?? '', { expiresIn: '1h' });
+  });
+  it('should sucessifully return users if a valid token is sent', async () => {
+    const response = await axios.post(
+      serverUrl,
+      { query: query.query },
+      {
+        headers: { 'Content-Type': 'application/json', Authorization: testToken },
+      },
+    );
+    const responseData = response.data.data.users;
+    expect(responseData).to.be.an('array');
+    expect(responseData).to.have.lengthOf(3);
+    expect(responseData[0]).to.have.all.keys('id', 'name', 'email', 'birthDate');
+    expect(responseData[0].name).to.equal('usuario1');
```

Comment:
poderia verificar todos os campos dos 3 usuários

Reply:
Adicionei um for que faz a verificação de todos os campos dos 3 usuários